### PR TITLE
Enable events in platform

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

CAPI - 1196: Merge Events Module from Internal - Part 2

### Description of the Change

The Jenkins files will include events module and start pushing events.
The files that will need to be updated are all the
jenkins-build-pack-*/Jenkinsfile.

### Benefits

1. Jenkins jobs for build packs for all types of services should start publishing events.
2. Adding DISABLE_EVENTS in config repo should stop publishing of events.

### CODE REVIEW

File: jenkins-build-pack-api/Jenkinsfile [hunk 1]
```diff
@@ -164,7 +157,9 @@ node() {
 		sh "cp serverless.yml serverless-prod.yml"
 		sh "cp swagger/swagger.json swagger/swagger-prod.json"
 
+		def context_map = [created_by : config['owner']]
 		stage('Pre-Build Validation') {
+			events.sendStartedEvent('VALIDATE_INPUT', 'pre-build validation starts', context_map, 'NA')
```
Comment (Deepak-Babu) [inline jenkins-build-pack-api/Jenkinsfile]: dont hardcode event names. Take it from enumeration.

**Comment by SiniWilson** [comment]
In cooperated the review comment

**Comment by suryajak** [comment]
This PR should be against vnext_events branch

**Comment by suryajak** [comment]
@SiniWilson : Can you please update the base branch to v1.3.1 and resubmit the PR?

**Pull request closed without merging**